daffodil-test/src/test/scala/org/apache/daffodil/usertests/TestSepTests.scala

```diff
@@ -70,6 +70,8 @@ class TestSepTests {
   @Test def test_sep_ssp_never_5(): Unit = { runner.runOneTest("test_sep_ssp_never_5") }
 
   // DAFFODIL-2791
-  @Test def test_treatAsMissing_occursIndex(): Unit = { runner.runOneTest("test_treatAsMissing_occursIndex") }
```

Comment:
This line doesn't seem too long to me. Fits fine on my screen for github code review and my browser is zoomed to 133%.

Should we allow longer lines?

Reply:
I'm using a laptop screen since I'm away from both home and office now, so I see only this much of the line and I would have to use the horizontal scrollbar to see the rest of the line (my browser is zoomed to 100%):

```scala
  @Test def test_treatAsMissing_occursIndex(): Unit = { runner.runOneTest("test_treatAsMissing_occursI
```

BTW, I checked and found out if I increase maxColumn to 100, sbt scalafmtCheckAll fails and scalafmtAll reformats a bunch of files to re-join some files that had been split before.  Any time we change the allowed length, we'll reformat some files.